src/main/cpp/output/VisualizerFile.h

```diff
@@ -0,0 +1,43 @@
+#ifndef VIS_FILE_H_INCLUDED
+#define VIS_FILE_H_INCLUDED
+/**
+ * @file
+ * Header for the VisualizerFile class.
+ */
+
+#include <fstream>
+#include <string>
+#include <vector>
+#include "VisualizerData.h"
+#include "core/Atlas.h"
+
+namespace stride {
+namespace output {
+
+/**
+ * Produces a file with the necessary data for visualization.
+ */
+class VisualizerFile
+{
+public:
+	/// Constructor: initialize.
+	VisualizerFile(const std::string& file = "stride_vis");
+
+	/// Destructor: close the file stream.
+	~VisualizerFile();
+
+	/// Print the given visualisation data.
+	void Print(Atlas::TownMap townMap, const VisualizerData& visualizer_data);
```

Comment:
Is an `Atlas::TownMap` a big data structure? If so, can you pass it by reference?